### ISSUE TITLE
cannot open param.lua: No such file or directory

### ISSUE THREAD

**Issue by PengfeiRen96** [mks0601/V2V-PoseNet_RELEASE#1]
db: NYU mode: train	
training data loading...	
testing data loading...	
invalid frame in test set: 0
/root/torch/install/bin/luajit: /root/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 6 callback] cannot open param.lua: No such file or directory

**Comment by mks0601** [comment]
I pushed modified code. solved!
